docs/connectors/building.mdx

```diff
@@ -3,19 +3,89 @@ title: "Building Connectors"
 sidebar_position: 5
 ---
 
-Conduit connectors can be built in any programming language that supports gRPC.
-To make it easier to write connectors we provide
-a [Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk) written in
-Go. Using the SDK is the recommended way of writing a Conduit connector.
+## Developing Custom Connectors
 
-## Conduit connector template
+The Conduit platform offers extensibility through the development of connectors, which can be written in any programming language that supports [gRPC](https://grpc.io/). For ease of development, a Go-based [Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk) is provided and its use is the recommended approach for creating Conduit connectors.
```

Comment:
I wouldn't call it "Conduit platform", to not create confusion with our platform offering. Let's stick to just "Conduit".

Reply:
@lovromazgon made the change with 97c7e7a65526430fc898e8ab16511ea777acc261